src/ConfigManager.py

```diff
@@ -76,17 +76,31 @@ def generate_benchmark_configs(self, app_modules: list[dict]) -> None:
         self.application = _get_instance_with_sub_options(app_modules, app_name)
 
         application_config = self.application.get_parameter_options()
-        application_config = ConfigManager._query_for_config(
-            application_config, f"(Option for {application_answer['application']})")
-
-        submodule_options = self.application.get_available_submodule_options()
-        submodule_answer = checkbox(key='submodules', message="What submodule do you want?",
-                                    choices=submodule_options)
+        more_app_config = {}
 
+        full_application_config = ConfigManager._query_for_config(
```

Comment:
@Marvmann I think this should be described in Read The Docs under the "Developer" section.

Reply:
Ok, I will provide a paragraph.